src/Arbitrary.hs

```diff
@@ -638,8 +639,9 @@ instance Arbitrary InitialPassword where
     shrink = gshrink
 
 instance Arbitrary EncryptedPassword where
-    arbitrary = garbitrary
-    shrink    = gshrink
+    arbitrary = mk <$> arb <*> arb
+      where
+        mk salt pass = ScryptEncryptedPassword . getEncryptedPass $ encryptPass' (Salt salt) (Pass pass)
```

Comment:
i wonder why you don't want the other two cases, but i don't feel strongly either way.